src/pcapi/routes/webapp/beneficiaries.py

```diff
@@ -116,5 +117,6 @@ def id_check_application_update(
         application_id = int(body.id)
     except ValueError:
         raise ApiErrors({"id": "Not a number"})  # pylint: disable=raise-missing-from
+    logger.info("[BENEFICIARIES] Received application id %s to process", application_id)
```

Comment:
```suggestion
    json_logger.info("[BENEFICIARIES] Received an application to process", extra={"applicationId": application_id})
```

Un format json avec un message static et du contexte simplifiera le calcul du nombre d'applications reçues. Ça évitera les recalculs pour faire des stats (genre : nombre de dossiers traités).